Remove generator settings dialog, put those properties in settings

vol0n opened the pull request:
- Now all settigns are in tools - UTBot Settings
- Fixed a bug when generator settings fields were not saved between
  launches of ide.

Closes #7

vol0n committed:
Remove generator settings dialog,put it in settigs

- Now all settigns are in tools - UTBot Settings
- Fixed a bug when generator settings fields were not saved between
  launches of ide.

vol0n committed:
Add callbacks on apply and reset for textfields

vol0n committed:
Fix paths policy, use chooseFiles

- Tests dir path - absolute path must be used
- File path - absolute path must be used
- When choosing source files in settings, now it is possible to select
several files at once.

PolyProgrammist commented on src/main/kotlin/com/github/vol0n/utbotcppclion/services/GeneratorSettings.kt:
```diff
@@ -1,30 +1,46 @@
+    val logger = Logger.getInstance(this::class.java)
+
+    init {
+        logger.info("constructor was called: project == $project")
```
Is this for debug purpose? If it is, delete it. If not, please specify what constructor was called

PolyProgrammist commented on src/main/kotlin/com/github/vol0n/utbotcppclion/services/GeneratorSettings.kt:
```diff
@@ -1,30 +1,46 @@
 
-    override fun loadState(p0: GeneratorSettings) {
+    override fun loadState(state: GeneratorSettings) {
+        logger.info("loadState was called: $state \n $this")
```
Specify it is in generatorsettings or delete it

PolyProgrammist replied:
And everywhere else

vol0n committed:
Add remote path setting when using docker

- When running utbot on docker with mount, the local paths should be
converted to corresponding docker paths, for this a path to project on
docker filesystem is needed
- Fix: when server generates tests: tests folder is not visible
  immediately.
- Add more logs for Client methods' calls.

vol0n committed:
Remove commented code

vol0n committed:
"Fix build"

- Update quadana version
- Add CL to verifier ide versions
